### PR TITLE
US1 // display partsList

### PR DESCRIPTION
**Pull request by matschi3**
completed US1 with essential styling.

- create dummy Partsdata
- display PartsData on FrontEnd
- create basic partCard style 

[US1](https://github.com/matschi3/capstone/issues/1)
[Deployment](https://capstone-c9w51mgjn-matschi3.vercel.app/)

Thanks for your time ✌🏼

### CODE REVIEW

**Comment by rene-fas** [comment]
Acceptance criteria
 - [x] user can see a list of all parts with basic partData -> looks fine to me
 - [x] listItems have same size -> looks fine to me aswell
 - [x] listItems width almost screenWidth -> margin does its task (might be worth some research if a relative margin (%) is 
        fitting in this case unfortunately i cant tell you but it works here.
 - [x] list scrolls vertically -> works fine
 - [ ] heading is always visible -> which heading? I cant see an implemented header and the title of the list is not always visible
 
 Mostly looks fine to me but i dont see an always visible header. Otherwise it looks good.

**Comment by Katemuya** [comment]
LGTM !
Very well written code that made it easy to read. I agree with rene-fs concerning the visible heading. I left a comment on your styling file. Once that is fixed I think you are good to go🚀

**Comment by rene-fas** [comment]
Header works. Looks good to me, meets your US criteria.

File: components/PartsList/PartsList.styled.js
[the remark this answers is not in the document]
Comment (matschi3) [reply components/PartsList/PartsList.styled.js:1]: Thanks for your review. Indeed i have position fixed now on h1. And moved it out of container.
👍🏼

**Comment by matschi3** [comment]
also thank you @rene-fas for your review, and the re-review again so i don't need to repost 👍🏼

**Pull request merged**